### DISCUSSION TITLE
SOLVED Kurzweil does not recognize ZuluSCSI

### DISCUSSION THREAD

**Discussion by hubweil** [ZuluSCSI/ZuluSCSI-firmware#274] [General]
SOLVED

Hello, i extra registered, because i'm stuck with the zuluSCSI in my Kurzweil K2500R.

i got the red zuluSCSI and a 128gb exFat formatted SD. 
i downloaded the "empty file" provided by one of the members here (thank you!) 
and copied it onto the SD. and put the SD in the zuluSCSI.

now i got 2 problems:

- when i plug the zuluSCSI in the "internal hd" socket in the Kurzweil, the Kurz won't boot. It powers up but display stays blue.
   i tried with zuluSCSI termination on and off. No difference. 
   when i plug it into the socket, that leads to the scsi connector on the back, then it powers up.

but even when it powers up (right now via socket for connctor on back) then no SCSI drive is recognized.
i also put another cdrom iso on the SD, named it HD01.hda, this too won't get recognized.

do i also have to format the SD in the Kurzweil? 
i tried but can't as it does not get recognized so far.

thank you for your help!

**Comment by hubweil** [comment]
EDIT:

**Reply by aperezbios** [reply]
@hubweil it would be helpful if you could describe what the resolution was, so others can benefit from the knowledge.

**Comment by hubweil** [comment]
Sure, sorry:

in my K2500R  the socket for the internal SCSI drive is reversed. I think this might also be the case in K2500 keyboard and K2600 models. The red stripes do not align. The solution was, to remove the nub/nose on the 50pin cable's plug with a box cutter (carefully, watch fingers and cables!) and flip it, then the red stripe is on the correct side and the ZuluSCSI / virtual drives are recognized.  BTW ZuluSCSI is great, it was a great improvement for my K2500R.
